docs/order-request.json

```diff
@@ -0,0 +1 @@
+{"name": "clipped-geom", "products": [{"item_ids": ["20220605_124027_64_242b"], "item_type": "psscene", "product_bundle": "analytic_sr_udm2"}]}
```

Comment:
was it intentional to upload this file? I didn't see it referenced. In general, I am a fan of using the repo to store files. Could help keep things clean by adding them to a data subdir of the docs folder.

Reply:
no, not intentional. But yeah, storing some of the sample requests directly in the repo would make sense to me.